### ISSUE TITLE
Update Slovak translations - aligned with Czech which is really similar

### ISSUE THREAD

**Issue by scepka** [squidfunk/mkdocs-material#6860]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "sk",
  "action.edit": "Upraviť túto stránku",
  "action.skip": "Preskočiť na obsah",
  "action.view": "Zobraziť zdroj tejto stránky",
  "announce.dismiss": "Už nezobrazovať",
  "blog.archive": "Archív",
  "blog.categories": "Kategórie",
  "blog.categories.in": "v",
  "blog.continue": "Pokračovať v čítaní",
  "blog.draft": "Návrh",
  "blog.index": "Späť na index",
  "blog.meta": "Metadata",
  "blog.references": "Súvisiace odkazy",
  "clipboard.copy": "Kopírovať do schránky",
  "clipboard.copied": "Skopírované do schránky",
  "consent.accept": "Prijať",
  "consent.manage": "Spravovať nastavenia",
  "consent.reject": "Odmietnuť",
  "footer": "Pätka",
  "footer.next": "Ďalej",
  "footer.previous": "Späť",
  "header": "Hlavička",
  "meta.comments": "Komentáre",
  "meta.source": "Zdroj",
  "nav": "Navigácia",
  "readtime.one": "1 min čítanie",
  "readtime.other": "# min čítanie",
  "rss.created": "RSS kanál",
  "rss.updated": "RSS zdroj aktualizovaného obsahu",
  "search": "Vyhľadávanie",
  "search.placeholder": "Hľadať",
  "search.share": "Zdieľaj",
  "search.reset": "Vyčistiť",
  "search.result.initializer": "Spúšťam hľadanie",
  "search.result.placeholder": "Pre vyhľadávanie začni písať",
  "search.result.none": "Žiadne vyhovujúce dokumenty",
  "search.result.one": "Vyhovujúci dokument: 1",
  "search.result.other": "Vyhovujúce dokumenty: #",
  "search.result.more.one": "1 ďalší na tejto stránke",
  "search.result.more.other": "# ďalších na tejto stránke",
  "search.result.term.missing": "Chýbajúci",
  "select.language": "Vyberte jazyk",
  "select.version": "Vyberte verziu",
  "source": "Zobraziť repozitár",
  "source.file.contributors": "Prispievatelia",
  "source.file.date.created": "Vytvorené",
  "source.file.date.updated": "Posledná aktualizácia",
  "tabs": "Karty",
  "toc": "Obsah",
  "top": "Na začiatok"
}[key] }}{% endmacro %}
```

### Country flag

:flag_sk:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks for providing the remaining translations! Added in e51d29d51.

**Comment by squidfunk** [comment]
Released as part of [9.5.13](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.13).
